Lib/test/test_pickle.py

```diff
@@ -699,6 +702,58 @@ def test_multiprocessing_exceptions(self):
                                  ('multiprocessing.context', name))
 
 
+class CommandLineTest(unittest.TestCase):
```

Comment:
If we have lots of CLI tests, we can have a `test.support.cli_helper` module that would group such classes. We can also have some methods responsible for checking invocation etc.

Reply:
I tried locally to see if we can a generic helper but no so let's leave it as is for now.